### PR TITLE
add alist

### PR DESCRIPTION
**Pull request by Sagit-chu**

### CODE REVIEW

**Review by Ornias1993 — CHANGES_REQUESTED** [review]
Please use a versioned tag and make sure to digest-pin it.
See other images for examples.

**Comment by Sagit-chu** [comment]
> Please use a versioned tag and make sure to digest-pin it. See other images for examples.

OK

File: mirror/alist/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM xhofe/alist:latest@sha256:5aaf09a89ce257292d29260bd3e857d1ebf79d7bcf94db2ce8e64d7c0743a57a
```
Comment (Ornias1993) [inline mirror/alist/Dockerfile:1]: Super that it's digest pinned, but we want versioned tags if they are available, not latest.

**Comment by Ornias1993** [comment]
Cool thanks!

**Pull request merged**